externs/shaka/player.js

```diff
@@ -706,7 +706,7 @@ shaka.extern.AdvancedDrmConfiguration;
  *   <code>{'com.widevine.alpha': 'https://example.com/drm'}</code>.
  * @property {!Object.<string, string>} clearKeys
  *   <i>Forces the use of the Clear Key CDM.</i>
- *   A map of key IDs (hex) to keys (hex).
+ *   A map of key IDs (hex or raw) to keys (hex or raw).
```

Comment:
I would not have understood "raw" to mean "base64-encoded".  I'm going to update this.